spec/features/user_answers_required_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+feature "user answers a question" do
+  let!(:admin) { FactoryGirl.create(:admin) }
+  let!(:survey1) { FactoryGirl.create(:user_survey, title: "something else", admin: admin) }
+  let!(:invisible_survey) { FactoryGirl.create(:survey, admin: admin) }
+  let!(:other_survey) { FactoryGirl.create(:user_survey, admin: admin, group: "different group") }
+  let!(:question2) { FactoryGirl.create(:question, survey: other_survey, text: true, required: true) }
+    # let!(:question5) { FactoryGirl.create(:question, survey: other_survey, rating: true, text: true, required: true) }
```

Comment:
Line is too long. [120/80]<br>Incorrect indentation detected (column 4 instead of 2).